tools/generate-templates.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rm -rf test/roles/* || true
+mkdir -p test/roles
+
+while IFS='' read -r line; do DRIVER_NAMES+=("$line"); done < <(python "tools/toml_to_json.py" "pyproject.toml" | jq -rc '.project."entry-points"."molecule.driver" | keys[]')
```

Comment:
why not making the python script outputting the list of drivers ?
This would avoid using jq and if we need later a way to get the list of drivers, the python script would be very handy.

Reply:
@apatard Yeah at first i wanted to do it without a script with some shell command but couldn't find any reliable python-cli toml parser.
I have updated it to output the plugins, it makes more sense this way